spring-cloud-contract-tools/spring-cloud-contract-gradle-plugin/src/main/groovy/org/springframework/cloud/contract/verifier/plugin/StubRunnerOptionsFactory.groovy

```diff
@@ -0,0 +1,28 @@
+package org.springframework.cloud.contract.verifier.plugin
+
+import groovy.transform.CompileStatic
+import org.springframework.cloud.contract.stubrunner.StubRunnerOptions
+import org.springframework.cloud.contract.stubrunner.StubRunnerOptionsBuilder
+import org.springframework.cloud.contract.stubrunner.spring.StubRunnerProperties
+
+@CompileStatic
+class StubRunnerOptionsFactory {
```

Comment:
Might want to make this `@PackageScoped` that way it doesn't become a part of the public API, huh @marcingrzejszczak?

Reply:
Done, thx.